src/compiler/crystal/compiler.cr

```diff
@@ -21,7 +21,7 @@ module Crystal
   # optionally generates an executable.
   class Compiler
     CC = ENV["CC"]? || "cc"
-    CL = "cl.exe"
+    CL = ENV["CC"]? || "cl.exe"
```

Comment:
I think the naming of these constants is a bit confusing when `CL` is `ENV["CC"]`. 
They are probably pretty unnecessary anyway. The single use is in `linker_command`. So it doesn't even make much sense to have a constant.

But I'm fine with merging this and refactor afterwards.